cfme/tests/control/test_basic.py

```diff
@@ -428,6 +440,24 @@ def test_assign_host_condition_to_host_policy(
     random_host_control_policy.assign_conditions()  # unassign
 
 
+@pytest.mark.tier(3)
+@pytest.mark.skipif(current_version() < "5.6", reason="requires cfme 5.6 and higher")
+def test_assign_events_to_container_image_control_policy(
+        random_container_image_control_policy,
+        container_image_condition,
+        soft_assert):
+    """This test checks whether an event is assigned to container image control policy.
+    Steps:
+        * Create a container image control policy.
+        * Assign an event to the created policy.
+    """
+    random_container_image_control_policy.assign_conditions(
+        container_image_condition)
+    soft_assert(random_container_image_control_policy.is_condition_assigned(
```

Comment:
If it checks only one single thing, it would be better to do:
```python
request.addfinalizer(random_container_image_control_policy.assign_conditions)
assert random_container_image_control_policy.is_condition_assigned(container_image_condition)
```